github3/github.py

```diff
@@ -273,6 +273,8 @@ def create_issue(self, owner, repository, title, body=None, assignee=None,
             formatted
         :param str assignee: (optional), Login of person to assign
             the issue to
+        :param assignees: (optional), login of the users to assign the
```

Comment:
Small English tweak/question. Would it be 'logins of the users' or is 'login of the users' the "correct" English here?